cfme/tests/cloud/test_targeted_refresh.py

```diff
@@ -1,14 +1,46 @@
+import fauxfactory
 import pytest
 
 from cfme import test_requirements
+from cfme.cloud.provider.ec2 import EC2Provider
+from cfme.utils.blockers import BZ
+from cfme.utils.generators import random_vm_name
+from cfme.utils.wait import wait_for
+
 
 pytestmark = [
     test_requirements.ec2,
+    pytest.mark.ignore_stream("upstream"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.provider(
+        [EC2Provider],
+        scope='module'
+    )
 ]
 
+DELAY = 15
+TIMEOUT = 1500
 
-@pytest.mark.manual
-def test_ec2_targeted_refresh_instance():
+
+def wait_for_power_state(vms_collection, instance_name, power_state):
+    wait_for(lambda: vms_collection.get(name=instance_name)["power_state"] == power_state, delay=15,
```

Comment:
Did you mean to use `Delay` here?

Reply:
Yes, I need to check it every `15` seconds.

Reply:
Ah I meant did you mean to use the constant `DELAY` you've defined on L21?

Reply:
Ahh, right! Changed locally.

Reply:
Changed in the PR.